python/datasources/acs_poverty.py

```diff
@@ -0,0 +1,333 @@
+        for table_name, df in self.frames.items():
+            # All breakdown columns are strings
+            column_types = {c: "STRING" for c in df.columns}
+
+            column_types[ABOVE_POVERTY_COL] = "INT64"
+            column_types[BELOW_POVERTY_COL] = "INT64"
+
+            gcs_to_bq_util.append_dataframe_to_bq(
+                df, dataset, table_name, column_types=column_types
+            )
+
+    # Uploads the acs data to gcs and returns if files are diff.
+    def upload_to_gcs(self, bucket):
+        file_diff = False
+        for group in POVERTY_BY_RACE_SEX_AGE_GROUPS:
+            for is_county in [True, False]:
+                file_diff = (
+                    url_file_to_gcs.url_file_to_gcs(
+                        self.base_url,
+                        get_params(group, is_county),
+                        bucket,
+                        get_filename(group, is_county),
+                    )
+                    or file_diff
+                )
+
+        return file_diff
+
+    # Write all the datas to local files to debug
+    def write_local_files_debug(self):
+
+        with open("acs_poverty_metadata.json", "w") as f:
+            print(json.dumps(self.metadata, indent=4), file=f)
+
+        self.getData()
+
+        with open("acs_poverty_internal_data.json", "w") as f:
+            print(
+                json.dumps({str(k): v for k, v in self.data.items()}, indent=4), file=f
+            )
+
+        self.split_data_frames()
+
+        for table_name, df in self.frames.items():
+            df.to_csv(table_name + ".csv", index=False)
+            df.to_json(table_name + ".json", orient="records", indent=4)
+
+    # Get the acs data for all the group / concept combinations.
+    # Accumulate to in memory data to be split later
+    def getData(self, gcs_bucket=None):
+        for group in POVERTY_BY_RACE_SEX_AGE_GROUPS:
+            for is_county in [True, False]:
+                data = None
+                if gcs_bucket is None:
+                    data = get_acs_data_from_variables(
+                        self.base_url, get_params(group, is_county)
+                    )
+                else:
+                    data = gcs_to_bq_util.load_values_as_json(
+                        gcs_bucket, get_filename(group, is_county)
+                    )
+                self.accumulate_acs_data(data)
+
+    """
+    Takes data in the form of
+    [
+        [C27001A_002E, C27001A_003E, C27001B_002E, C27001B_002E][state][?county],
+        [12345, 1245, 123546, 124567, 01, 02]
+        ...
+    ]
+    This method determines the variables at the top (aka: key_row)
+    matches the value with the metadata from the prefix_suffix list
+    and stores it in the self.data as a tuple
+
+    (state_fip, county_fip, age, sex, race) example:
+    {
+        "('01', 'None', '0-6', 'White', $0-$10000)": {
+            "Population": "143",
+
+        "('01', 'None', '0-6', 'White', $10,000-$20,000)": {
```

Comment:
Is this example in the comment still accurate despite this being poverty data rather than income data?